sdk/core/core-tracing/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/core-tracing",
-  "version": "1.0.0-preview.14",
+  "version": "1.0.0-preview.13",
```

Comment:
TODO: I just do this to make it easy to test, but I'll return this to preview.14 before merging of course